docs/projects/tcet-linux-website/frontend-components/download-section.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Download Section
+id: download-section
+description: Download Component in TCET Linux Website
+sidebar_label: Download Section
+keywords: [Download link, Links, Downloads, Installers, DownloadSection]
+---
+
+The Download component represents the Second section of the [**TCET Linux website**](https://linux.tcetmumbai.in/). It is one of the most crucial component as it provides important information and actions related to downloading and verifying the TCET Linux software. This section serves as a gateway for users to access the software and its associated resources. The page likely presents data on the **Download Availability**, the **Verification** and **Additional Resources**. The section informs users that the TCET Linux source code and installers are available for download. It provides direct download links to the software, allowing users to easily obtain the necessary files
+
+:::tip
+You can visit ***[DownloadSection.astro](https://github.com/tcet-opensource/tcet-linux-website/blob/main/src/components/DownloadSection.astro)***
+:::
+
+:::info
+
+   1. We have created a division using div tag with the id as download which would help to deliver this component of code when requested.
+   2. Then inside this we have used react to add gradient background as per requirments and pointer interaction is set as none.
+   3. Then inside this component we have made two column using div tag for left and right columns.
+   4. Here in the `left column` we have added downlodable links for iso file and checksum file also verify and istallation video tutorial is also provided.
+   5. Lastly in the `right column` we have added ***an image of a man with a laptop*** for beautification. 
+:::
+
+## Download Section & Gradient
+
+**Here's a breakdown of the code to understand it's functionality:**
+
+1. This code represents the start of the DownloadSection component. The div element with the id attribute `download` serves as a container for the entire section. It has classes for styling, such as a full width, background color, relative positioning, and hidden horizontal overflow.
+ 
+     ```jsx title="DownloadSection.astro" {1} showLineNumbers
+     <div  id="download"  class="w-full bg-slate-800 relative overflow-x-hidden px-10 lg:px-12">
+     // Furthur components were added 
+     </div>
+     ```
+
+2. This div element adds a `gradient background` to the section. It has the class `downloadSectionGradient` and the style `pointer-events-none` to prevent any interaction with the element.
```

Comment:
When u mention `<div>` or `<p>` of the codes highlight it using code block 
Change this in whole document